packages/react-dom/src/__tests__/ReactDOMServerIntegrationReconnecting-test.js

```diff
@@ -354,8 +354,8 @@ describe('ReactDOMServerIntegration', () => {
           <div>{''}</div>,
         ));
 
-      it('can explicitly ignore reconnecting more children', () =>
-        expectMarkupMatch(
+      it('can not ignore reconnecting more children', () =>
+        expectMarkupMismatch(
```

Comment:
Should we duplicate these tests so that we still have a test for the legacy API? We already kept tests in other places for legacy API.

Reply:
Good point, added back